src/Validation/Validation.php

```diff
@@ -522,15 +522,47 @@ public static function localizedTime($check, $type = 'datetime', $format = null)
 
     /**
      * Boolean validation, determines if value passed is a boolean integer or true/false.
+     * The list of what is considered to be boolish values, may be set via $booleanValues.
      *
      * @param string $check a valid boolean
+     * @param string $booleanValues list of valid boolish values, defaults to `[true, false, 0, 1, '0', '1']`.
      * @return bool Success
      */
-    public static function boolean($check)
+    public static function boolean($check, array $booleanValues = [])
```

Comment:
You mean because someone could have overwritten it? I do agree. I can take this out of this PR and set it up as a PR towards 4.x if you prefer. I don't mind.

Else I am also fine for this to go into 3.4 and I can patch the migration docs, I guess. Let me know how to processed once you have a stance @ core-team.